students/rick_halsell/lesson02/Fizz_Buzz_Exercise.py

```diff
@@ -0,0 +1,11 @@
+def FizzBuzz():
+    for i in range(1,101):
+        if i%3 ==0 and i%5 == 0:
```

Comment:
You'll want to be consistent with your spacing here: if i % 3 == 0 etc.  Here and below.